css.md

```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
 - Colors: (use generic variable names like recent cfgov-refresh update)
 - Breakpoints: standard variable names for commonly used breakpoints
 
+### Comments
+
+### JSDocs
+[Use JSDocs](http://usejsdoc.org/) style comments for comment blocks and `//` for inline comments.
+
+**Example**
+
+```less
+/**
+ * Default button styles
+ * @class .btn
+ * @elements a, button
+*/
+.btn {
+    display: inline-block;
+
+  &:focus {
+    background-color: @btn-bg-hover;
+    outline: 1px dotted @btn-bg;
+    // outline-offset is not supported everywhere but it adds a nice touch where it is
+    outline-offset: 1px;
+  }
```

Comment:
Indentation here is inconsistent between `display` property and `&:focus` selector.